library/caddy

```diff
@@ -1,60 +1,60 @@
 # this file is generated with gomplate:
 # template: https://github.com/caddyserver/caddy-docker/blob/6355eb99a01d12c55a7e043129bda9cb90da81d6/stackbrew.tmpl
-# config context: https://github.com/caddyserver/caddy-docker/blob/6355eb99a01d12c55a7e043129bda9cb90da81d6/stackbrew-config.yaml
+# config context: https://github.com/caddyserver/caddy-docker/blob/ca397869945278d48511fef1fcdb9e0fac8d9e1f/stackbrew-config.yaml
 Maintainers: Dave Henderson (@hairyhenderson)
 
-Tags: 2.1.0-beta.1-alpine
-SharedTags: 2.1.0-beta.1
+Tags: 2.1.0-alpine, 2-alpine, alpine
+SharedTags: 2.1.0, 2, latest
 GitRepo: https://github.com/caddyserver/caddy-docker.git
 Directory: 2.1/alpine
-GitCommit: 4d9226a81a17ff7da5b3976550f9c2e0b8d61aad
+GitCommit: 89777d071050735e2769ea3a48d0325adac96900
 Architectures: amd64, arm64v8, arm32v6, arm32v7, ppc64le, s390x
 
-Tags: 2.1.0-beta.1-builder
+Tags: 2.1.0-builder, 2-builder, builder
 GitRepo: https://github.com/caddyserver/caddy-docker.git
 Directory: 2.1/builder
-GitCommit: 6355eb99a01d12c55a7e043129bda9cb90da81d6
+GitCommit: 89777d071050735e2769ea3a48d0325adac96900
 Architectures: amd64, arm64v8, arm32v6, arm32v7, ppc64le, s390x
 
-Tags: 2.1.0-beta.1-windowsservercore-1809
-SharedTags: 2.1.0-beta.1-windowsservercore, 2.1.0-beta.1
+Tags: 2.1.0-windowsservercore-1809, 2-windowsservercore-1809, windowsservercore-1809
+SharedTags: 2.1.0-windowsservercore, 2-windowsservercore, windowsservercore, 2.1.0, 2, latest
 GitRepo: https://github.com/caddyserver/caddy-docker.git
 Directory: 2.1/windows/1809
-GitCommit: 6355eb99a01d12c55a7e043129bda9cb90da81d6
+GitCommit: 89777d071050735e2769ea3a48d0325adac96900
 Architectures: windows-amd64
 Constraints: windowsservercore-1809
 
-Tags: 2.1.0-beta.1-windowsservercore-ltsc2016
-SharedTags: 2.1.0-beta.1-windowsservercore, 2.1.0-beta.1
+Tags: 2.1.0-windowsservercore-ltsc2016, 2-windowsservercore-ltsc2016, windowsservercore-ltsc2016
+SharedTags: 2.1.0-windowsservercore, 2-windowsservercore, windowsservercore, 2.1.0, 2, latest
 GitRepo: https://github.com/caddyserver/caddy-docker.git
 Directory: 2.1/windows/ltsc2016
-GitCommit: 6355eb99a01d12c55a7e043129bda9cb90da81d6
+GitCommit: 89777d071050735e2769ea3a48d0325adac96900
 Architectures: windows-amd64
 Constraints: windowsservercore-ltsc2016
 
-Tags: 2.0.0-alpine, 2-alpine, alpine
-SharedTags: 2.0.0, 2, latest
+Tags: 2.0.0-alpine
```

Comment:
Is the 2.0.x series going to continue to be supported as well?

Removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Reply:
@tianon that was my plan at least short-term, but I'll double-check with @mholt 🤔

Reply:
Our `master` is on the 2.2.x dev tree already; I'm not sure why we'd need to keep supporting 2.0.

Reply:
ok, fair enough - I'll remove 2.0!